pkg/pxc/users/users.go

```diff
@@ -55,6 +55,24 @@ func (u *Manager) UpdateUsersPass(users []SysUser) error {
 				}
 				return errors.Wrap(err, "update password")
 			}
+			if user.Name == "monitor" {
+				_, err = tx.Exec("GRANT SELECT, PROCESS, SUPER, REPLICATION CLIENT, RELOAD, CREATE USER ON *.* TO 'monitor'@?", host)
```

Comment:
why CREATE USER is needed?

Reply:
removed https://github.com/percona/percona-xtradb-cluster-operator/pull/450/commits/abad5844a306f8953bacde563dbec1608caf9603